src/components/Tasks/TaskEdit.tsx

```diff
@@ -88,6 +94,7 @@ class TaskEdit extends React.PureComponent<PropsType> {
       this.state.uid = event.uid;
       this.state.title = event.title ? event.title : '';
       this.state.status = event.status;
+      this.state.priority = event.priority || TaskPriorityType.Undefined;
```

Comment:
This should be `??` not `||`. I know it works (because 0 happens to be falsey so it too just maps to undefined), but still better to use `??` so we put what we mean.